new_arrivals_chi/migrations/versions/2024-04-25-2341_16cf46fcdfbf_initial_migration.py

```diff
@@ -0,0 +1,168 @@
+"""Initial migration
```

Comment:
🔬  Since you don't have any other migrations in your main branch, for the sake of clarity, I think it make be a good move to delete all the current migrations and create just one. That way the initial state of the DB is clear.